package.json

```diff
@@ -1,11 +1,12 @@
 {
-  "name": "debugger",
-  "displayName": "temporal-debugger-extension",
-  "description": "Temporal VSCode Debugger extension",
+  "name": "temporal-vscode-extension",
+  "displayName": "Temporal VS Code extension",
```

Comment:
```suggestion
  "displayName": "Temporal.io",
```

Or `Temporal Tools`? Displaying "VS Code extension" on the marketplace feels unnecessary, since everything is a vsc extension. 

<img width="426" alt="image" src="https://user-images.githubusercontent.com/251288/235714087-6ee5292d-1e83-4bad-8384-a170612a35ec.png">

Reply:
Very true 😅 What do you think about `Temporal.io Debugger`?